gamla/functional_generic.py

```diff
@@ -640,8 +655,11 @@ async def apply_spec_async(*args, **kwargs):
     )
 
 
-# Stacks functions on top of each other, so will run pairwise on the input.
-# Similar to juxt, only zips with the incoming iterable.
+#: Stacks functions on top of each other, so will run pairwise on the input.
+#: Similar to juxt, only zips with the incoming iterable.
```

Comment:
```suggestion
Construct a function apply the i'th function in an iterable to the i'th element of an iterable given

Note: Number of functions given should be equal to number of arguments given
```

I think "stacks on top" gives a wrong impression (as if the functions are composed or anything)